notion_assistant/notion_assistant.py

```diff
@@ -0,0 +1,29 @@
+# notion-assistant.py
+
+
+from pathlib import Path
+from notion.client import NotionClient
+from notion.block.basic import *
+
+
+secret = Path('C:\\Users\Роман\\Documents\\my_secret\\secret.txt').read_text().strip()
```

Comment:
let's have relative path to token instead of absolute
create subfolder 'config' and add file 'notion.token' (so that git automatically ignores that due to gitignore)
token_path = Path('') / 'config' / 'notion.token'
token = token_path.read_text().strip()